packages/terra-form-select/docs/SelectField.md

```diff
@@ -0,0 +1,30 @@
+# Terra Form Select Field
+
+A convenience wrapper assembling a [terra-form-select](https://github.com/cerner/terra-core/tree/master/packages/terra-form-select) within a [terra-form-field](https://github.com/cerner/terra-core/tree/master/packages/terra-form-field).
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-select/lib/SelectField`
```

Comment:
Does this work? Or should this just be `npm install terra-form-select`? then use via `import SelectField from 'terra-form-select/lib/SelectField`?

Reply:
Whoops, no this would not work. This has been updated